packages/portal-proto/src/components/Modals/SetModals/SetTable.tsx

```diff
@@ -56,7 +57,7 @@ const SelectCell: React.FC<SelectCellProps> = ({
           />
         ) : (
           <Radio
-            data-testid="radio-select-set"
+            data-testid={`radio-${set[1]}`}
```

Comment:
Any reason you are accessing the second element?

Reply:
Yeah it let's me put in the set name as part of the ID

Reply:
Updated to setName to be more descriptive